app/views/ServiceConfigPage.scala.html

```diff
@@ -46,7 +48,7 @@ <h1 id="service-header">Config: @serviceName</h1>
                     <div id="header-0" class="grid-item sticky-column truncate-content" style="width:300px; text-indent: 1.2em">Key</div>
                     <div class="gutter gutter-horizontal"></div>
                     @for((env, index) <- ConfigEnvironment.values.zipWithIndex) {
-                        <div id="header-@{index + 1}" class="grid-item truncate-content column-@env.asString" style="width:500px;">@env.asString</div>
+                        <div id="header-@{index + 1}" class="grid-item truncate-content column-@env.asString @if(!deployments.exists(_.environment.asString == env.asString) && env.asString != "local") {hidden} else{}" style="width:500px;">@env.asString</div>
```

Comment:
If the types are the same, you could do `deployments.exists(_.environment == env)`
I don't think the `else {}` is needed.

Reply:
One is an Environment and the other is a ConfigEnvironment